AppCenter/AppCenterTests/MSChannelUnitDefaultTests.m

```diff
@@ -540,6 +541,62 @@ - (void)testLogsSentWithSuccess {
   [responseMock stopMocking];
 }
 
+- (void)testDelegateDeadlock {
+
+  // If
+  NSObject *lock = [NSObject new], *syncCallback = [NSObject new], *syncBackground = [NSObject new];
+  [self initChannelEndJobExpectation];
+  id<MSLog> mockLog1 = [self getValidMockLog];
+  id<MSLog> mockLog2 = [self getValidMockLog];
+  id delegateMock = OCMProtocolMock(@protocol(MSChannelDelegate));
+  OCMStub([delegateMock channel:self.sut didPrepareLog:OCMOCK_ANY internalId:OCMOCK_ANY flags:MSFlagsDefault])
+      .andDo(^(__unused NSInvocation *invocation) {
+        // Notify that didPrepareLog has been called.
```

Comment:
Add a new line before comment.

Reply:
It's the first line within a block so unfortunately our code formater won't accept the new line.